jetstream/cli.py

```diff
@@ -56,28 +54,79 @@ class AllType:
 
 class ExecutorStrategy(Protocol):
     project_id: str
-    dataset_id: str
 
-    def __init__(self, project_id: str, dataset_id: str, *args, **kwargs) -> None:
+    def __init__(self, project_id: str, *args, **kwargs) -> None:
```

Comment:
I think we can add this back to the signature:

```suggestion
    def __init__(self, project_id: str, dataset_id: str, *args, **kwargs) -> None:
```